README.md

```diff
@@ -185,6 +192,20 @@ To create an additional nodegroup, use:
 eksctl create nodegroup --cluster=<clusterName> [--name=<nodegroupName>]
 ```
 
+Additionally, you can use the same config file used for create
+cluster:
```

Comment:
This should probably be `create cluster` or `eksctl create cluster`.

Reply:
Sure, okay